docs/_index.yaml

```diff
@@ -19,55 +19,101 @@ description: >
-      description: >
-        <h3>qsim</h3>
+    - description: >
+        <h4>qsim</h4>
         <p>
-        qsim is a Schrödinger full state-vector simulator. It computes all the
-        <em>2<sup>n</sup></em> amplitudes of the state vector, where <em>n</em>
-        is the number of qubits. Essentially, the simulator performs
-        matrix-vector multiplications repeatedly. One matrix-vector
-        multiplication corresponds to applying one gate. The total runtime is
-        proportional to <em>g2<sup>n</sup></em>., where <em>g</em> is the number
-        of 2-qubit gates. To speed up the simulator, we use gate fusion,
-        single-precision arithmetic, AVX/FMA instructions for vectorization and
-        OpenMP for multi-threading.
+        qsim is a full wave function simulator written in C++. It uses gate
+        fusion, AVX/FMA vectorized instructions and multi-threading using
+        OpenMP to achieve state of the art simulations of quantum circuits.
+        qsim is integrated with Cirq and can be used to run simulations of up
+        to 40 qubits on a 90 core Intel Xeon workstation.
         </p>
-        <h3>qsimh</h3>
+        <h4>qsimh</h4>
         <p>
-        qsimh is a hybrid Schrödinger-Feynman simulator. The lattice is split
-        into two parts and the Schmidt decomposition is used to decompose
-        2-qubit gates on the cut. If the Schmidt rank of each gate is <em>m</em>
-        and the number of gates on the cut is <em>k</em> then there are
-        <em>m<sup>k</sup></em> paths. To simulate a circuit with fidelity one, one
-        needs to simulate all the <em>m<sup>k</sup></em> paths and sum the results. The
-        total runtime is proportional to <em>(2<sup>n<sub>1</sub></sup> + 2<sup>n<sub>2</sub></sup>)m<sup>k</sup></em>,
-        where <em>n<sub>1</sub></em> and <em>n<sub>2</sub></em> are the qubit
-        numbers in the first and second parts. Path simulations are independent
-        of each other and can be trivially parallelized to run on supercomputers
-        or in data centers. Note that one can run simulations with fidelity <em>F < 1</em>
-        just by summing over a fraction <em>F</em> of all the paths.
+        qsimh is a hybrid Schrödinger-Feynman simulator built for parallel
+        execution on a cluster of machines. It produces amplitudes for user-
+        specified output bitstrings.
         </p>
+      buttons:
+      - label: Get started with qsim on Cirq
+        path: /qsim/tutorials/qsimcirq
+        classname: button button-primary
+        icon_name: open_in_new
+      - label: GitHub repository
+        path: //github.com/quantumlib/qsim
+        classname: button button-white
+        icon_name: open_in_new
+      code_block: |
+        <pre class = "prettyprint">
+        import cirq
+        import qsimcirq
+
+        # Pick up to ~36 qubits to run on a large multi-core Intel CPU machine
+        qubits = [cirq.GridQubit(i,j) for i in range(6) for j in range(6)]
```

Comment:
36 qubits is pretty ambitious for an example - even on a powerful machine, it may take several minutes to run. Another consideration: for general-purpose Colab machines, ~28 qubits is the maximum that fits in memory.

If we want to keep the large example, maybe add a line recommending smaller qubit counts for test runs?